### DISCUSSION TITLE
patchClientRoutes 动态路由问题

### DISCUSSION THREAD

**Discussion by Arima-s** [umijs/umi#12626] [General]
按照官方的文档，或者参照 routes数据结构，动态添加的路由，渲染不出指定页面
![20240807-172051](https://github.com/user-attachments/assets/9dd009a8-b415-4907-944c-4df8883e49a1)
![20240807-173214](https://github.com/user-attachments/assets/cf09ea0a-c2e1-4a90-a0a4-640c2b3a86f2)

😂 该怎么搞啊     

背景：
1、侧边栏菜单，是主应用渲染的，目前写死在主应用，想改造成 接口下发。
2、尝试在patchClientRoutes内改造routes，按照部分issues内的示例方法，行不通。 主应用 "@umijs/max": "4.3.10"

**Comment by bbuugg** [comment]
搞定了吗？也遇到了
